botplugins/quote.py

```diff
@@ -176,6 +176,29 @@ async def recall_quote(event):
         pass
 
 
+@borg.on(events.NewMessage())
+async def random_chance_quote(event):
+    if randint(0, 1000) != 69:
+        return
```

Comment:
Nice meme

However it would probably be best to use the `chance()` decorator instead:
https://github.com/Qwerty-Space/uniborg/blob/tanya/uniborg/util.py#L49-L59

Reply:
Maybe the chance should be lowered to 1/10000 or even 1/100000 so it doesn't get annoying, your groups tend to get 1000 messages in a short time.

Reply:
Maybe 5000